hermes/views.py

```diff
@@ -237,21 +290,22 @@ def get(self, request, *args, **kwargs):
         
         {title: <Title of the message>,
          topic: <kafka topic to post message to>, 
-         author: <submitter of the message>,
+         submitter: <submitter of the message>,
+         authors: <Text full list of authors on a message>
          message_text: <Text of the message to send>,
          event_id: <ID of the non-localized event for these candidates>,
-         data: {authors: <Text full list of authors on a message>,
-                main_data: {[{candidate_id: <ID of the candidate>,
-                              ra: <Right Ascension in hh:mm:ss.ssss or decimal degrees>,
-                              dec: <Declination in dd:mm:ss.ssss or decimal degrees>,
-                              discovery_date: <Date/time of the candidate discovery>,
-                              telescope: <Discovery telescope>,
-                              instrument: <Discovery instrument>,
-                              band: <Wavelength band of the discovery observation>,
-                              brightness: <Brightness of the candidate at discovery>,
-                              brightness_error: <Brightness error of the candidate at discovery>,
-                              brightness_unit: <Brightness units for the discovery, 
-                                  current supported values: [AB mag, Vega mag]>
+         data: {[{candidate_id: <ID of the candidate>,
```

Comment:
Calling it a candidate is a bit science-specific, and I wonder if other users might find it to be not applicable to their science?  More general label might be "Source ID" or similar?   Or perhaps just generalize the description instead